src/utils/MySQLDataHandler.py

```diff
@@ -43,6 +43,66 @@ def base64decode(str_val):
     return base64.b64decode(str_val.encode("utf-8")).decode("utf-8")
 
 
+def default_vc_entries(config):
```

Comment:
Can you add some test cases for this one?

Reply:
sure, I have some testcases. would discuss unitest details and put them in next PR.